HCDevice.py

```diff
@@ -92,6 +92,48 @@ def parse_values(self, values):
 
         return result
 
+    # Based on PR submitted https://github.com/Skons/hcpy/pull/1
+    def test_program_data(self, data):
+        if "program" not in data:
+            raise Exception("{self.name}. Message data invalid, no program specified.")
+
+        if isinstance(data["program"], str):
```

Comment:
in my opinion you can make the if shorter if you catch the ValueError:
`
>>> bla="blub"
>>> if isinstance(bla, str):
...   print("True")
...
True
>>> if isinstance(int(bla), int):
...   print("is int")
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: 'blub'
>>> bla="1111"
>>> if isinstance(int(bla), int):
...   print("is int")
...
is int
>>>`

Can you check this?